packages/bolt-storage-file/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import { ConversationStore } from '@slack/bolt';
+import encode from 'encoding-down';
+import leveldown from 'leveldown';
+import levelup, { LevelUp } from 'levelup';
+
+/**
+ * File implementation of Bolt ConversationStore
+ */
+export default class FileStore<ConversationState = unknown>
+  implements ConversationStore<ConversationState> {
+  private db: LevelUp;
+
+  constructor(path = './.db') {
+    this.db = levelup(encode(leveldown(path), { valueEncoding: 'json' }));
+  }
+
+  async set(
+    conversationId: string,
+    value: ConversationState,
+    expiresAt?: number,
+  ): Promise<void> {
+    await this.db.put(conversationId, { value, expiresAt });
+  }
+
+  async get(conversationId: string): Promise<ConversationState> {
+    const entry = await this.db.get(conversationId);
+
+    if (entry.expiresAt !== undefined && Date.now() > entry.expiresAt) {
+      // release the memory
+      await this.db.del(conversationId);
+
+      throw new Error('Conversation expired');
+    }
+
+    return entry.value;
+  }
```

Comment:
shouldn't we also error if the entry isn't found? bolt does this. 

https://github.com/slackapi/bolt/blob/master/src/conversation-store.ts#L39

Reply:
We should. Here's the test case https://github.com/IBM/slack-wrench/pull/2/files#diff-4b729154570b2a7cb86c2298a52b5eb3R53

They had to code it in by hand b/c memory store. level does it for us.